compose/bin/setup-install

```diff
@@ -37,16 +37,18 @@ bin/clinotty bin/magento setup:install \
   --amqp-user="$RABBITMQ_DEFAULT_USER" \
   --amqp-password="$RABBITMQ_DEFAULT_PASS" \
   --amqp-virtualhost="$RABBITMQ_DEFAULT_VHOST" \
-  --cache-backend=redis \
-  --cache-backend-redis-server="$REDIS_CACHE_BACKEND_SERVER" \
-  --cache-backend-redis-db="$REDIS_CACHE_BACKEND_DB" \
-  --page-cache=redis \
-  --page-cache-redis-server="$REDIS_PAGE_CACHE_SERVER" \
-  --page-cache-redis-db="$REDIS_PAGE_CACHE_DB" \
-  --session-save=redis \
-  --session-save-redis-host="$REDIS_SESSION_SAVE_HOST" \
-  --session-save-redis-log-level=4 \
-  --session-save-redis-db=2 \
+  --cache-backend="$CACHE_BACKEND" \
+  --cache-backend-redis-server="$CACHE_BACKEND_REDIS_SERVER" \
+  --cache-backend-redis-db="$CACHE_BACKEND_REDIS_DB" \
+  --page-cache="$PAGE_CACHE" \
+  --page-cache-redis-server="$PAGE_CACHE_REDIS_SERVER" \
+  --page-cache-redis-db="$PAGE_CACHE_REDIS_DB" \
+  --session-save="$SESSION_SAVE" \
+  --session-save-redis-host="$SESSION_SAVE_REDIS_HOST" \
+  --session-save-redis-log-level="$SESSION_SAVE_REDIS_LOG_LEVEL" \
+  --session-save-redis-db="$SESSION_SAVE_REDIS_DB" \
+  --session-save-redis-disable-locking="$SESSION_SAVE_REDIS_DISABLE_LOCKING" \
+  --search-engine=opensearch \
```

Comment:
Adding `opensearch` shouldn't be needed as this is a default engine in 2.4.6. It would have to be explicit if you would want to switch back to `elasticsearch7`